radix-engine-tests/tests/clock.rs

```diff
@@ -157,11 +156,10 @@ fn test_clock_comparison_methods_against_the_current_time() {
     // Act
     let manifest = ManifestBuilder::new()
         .lock_fee(test_runner.faucet_component(), 10.into())
-        .call_function(
-            package_address,
-            "ClockTest",
-            "set_current_time",
-            manifest_args!(CLOCK, 1669663688996 as i64),
+        .call_method(
+            CLOCK,
```

Comment:
I assume we should delete `ClockTest`'s `set_current_time` method?

Reply:
yup good point